src/theme/colors.ts

```diff
@@ -20,6 +20,8 @@ export const SECONDARY = {
   selected: '#b8ffef63',
   drawer_background: 'rgba(8, 188, 184, 0.04)',
   error_hover: 'rgba(198, 40, 40, 0.08)',
+  caregiver_profile_background: 'rgba(255, 255, 255, 1)',
```

Comment:
Why we need this color here? I think it's same as PRIMARY.light_main or quite similar, i couldn't find a difference from image

Reply:
I also don’t see any differences between them, but I took this color from the figma. I deleted it.